emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -50,7 +56,8 @@
   @Nullable OnEmojiPopupDismissListener onEmojiPopupDismissListener;
 
   final ViewTreeObserver.OnGlobalLayoutListener onGlobalLayoutListener = new ViewTreeObserver.OnGlobalLayoutListener() {
-    @Override public void onGlobalLayout() {
+    @Override
```

Comment:
should be on the same line just as it was previosly

Reply:
done